charts/lagoon-logging/values.yaml

```diff
@@ -251,6 +251,10 @@ enableDefaultForwarding: true
 # Set how many fluentd pods should be running
 fluentdReplicaCount: 3
 
+fluentdTolerations: []
```

Comment:
Could we add a comment on these new fields describing what they do and why you might want to use them? e.g. similar to the `fluentbitTolerations` comment.